### ISSUE TITLE
fix: add footer to home

### ISSUE THREAD

**Issue by oSumAtrIX** [ReVanced/revanced-website#175]
# Issue

Currently, every page except the home page has a footer. This is unconventional and may also impact SEO.

# Solution

Add the same footer to the home page as the other pages too.

# Additional context

It may be the case that the footer has not been added to the home page for some design-specific reasons. In this case, a solution has to be found, as it is unconventional not to include the footer and impacts SEO.

**Comment by KTibow** [comment]
I don't believe this is a problem for SEO or for users. Most content in the footer can be found on the home page already. The only things missing are:
- two socials (twitter and youtube - I don't think we really need to list them)
- an email (again, I don't think this needs to be on the home page when you can find it on the download page)
- this blurb of text:
  > ReVanced was born out of Vanced's discontinuation and it is our goal to continue the legacy of what Vanced left behind. Thanks to ReVanced Patcher, it's possible to create long-lasting patches for nearly any Android app. ReVanced's patching system is designed to allow patches to work on new versions of the apps automatically with bare minimum maintenance.

  which I think could be integrated into the home page's copy, given the current copy isn't very descriptive.

**Comment by oSumAtrIX** [comment]
PR https://github.com/ReVanced/revanced-website/pull/214 has been closed as it failed regarding the design (Particularly, the wave could have blended better with the footer). For that reason, when we decide to go forward with this issue, we will need to figure out a solution in terms of design. An initial proposal from my end is to make the wave appear at the bottom, just like it does at the top. This would prevent the "cutoff"

**Comment by KTibow** [comment]
I don't think this will be easy to integrate without messing with the footer or homepage, given that we currently have the screenshot being cut off at the bottom. Either we unnaturally don't show the rest of the screenshot, change how the screenshot is shown, or make the footer only use the space to the left of the screenshot.
The "image" link you sent must have expired, because it doesn't work.

**Comment by oSumAtrIX** [comment]
@KTibow I am not sure if I understood you correctly. Here is a new image link https://github.com/ReVanced/revanced-website/pull/214#pullrequestreview-1804022216

**Comment by KTibow** [comment]
This is what the home page looks like on my laptop:
![image](https://github.com/ReVanced/revanced-website/assets/10727862/32f5afea-3e80-4025-9412-007c8f6619a1)
You can see that if there was a theoretical footer, the footer would intersect with it.

**Comment by oSumAtrIX** [comment]
Yes. As part of my last comment that referred to this issue, I have added an initial design idea that could potentially solve it or act as a basis for better ideas and solutions that are yet to be found, or used until a solution is present. Do I understand that you have a question in regards to that?

**Comment by KTibow** [comment]
Even if there was a wave, the screenshot would still go on top of the wave and overlap the content. The main solution I see is to make the screenshot go under the wave.

**Comment by oSumAtrIX** [comment]
The bottom wave as I imagine it would take more height than the image so that the image does not overlap with the footer, alternative a scroll event can percentually shift the image up
